lib/_http_outgoing.js

```diff
@@ -223,9 +218,7 @@ OutgoingMessage.prototype._renderHeaders = function _renderHeaders() {
   const headers = {};
 
   if (headersMap !== null) {
-    const keys = ObjectKeys(headersMap);
-    for (var i = 0, l = keys.length; i < l; i++) {
-      const key = keys[i];
+    for (const key of ObjectKeys(headersMap)) {
```

Comment:
We should not make this change for performance reasons.

Reply:
resolved in https://github.com/nodejs/node/pull/30958/commits/17049d56621869d8776a922d5e56b6c5ec35be7b